lib/amazon_product_api/http_client.rb

```diff
@@ -46,54 +48,56 @@ def get(http: HTTParty)
       http.get(url)
     end
 
-
     private
 
-
     attr_reader :query, :page_num, :aws_credentials
 
+    # rubocop:disable UnneededDisable, Blank
```

Comment:
I'm guessing rubocop balked at this?

Reply:
Yeah, I had a comment here but silly me did it as a review and the comment went away (didn't copy/paste) when I canceled it.

Basically, the style guide prefers `if foo.blank?` to `unless foo.present?` however, when simply flipping that, the code wasn't throwing an expected error when it should have. I added the inline disable to get around that.